packages/grid/_modules_/grid/hooks/features/useApiRef.ts

```diff
@@ -12,5 +11,5 @@ export function useApiRef(): ApiRef {
   const logger = useLogger('useApiRef');
   logger.debug('Initializing grid api with EventEmitter.');
 
-  return React.useRef<GridApi>(new EventEmitter());
+  return React.useRef(new EventEmitter() as GridApi);
```

Comment:
why not use the Generic?

Reply:
Does it implement the EventEmitter type used in CoreApi?

Reply:
And btw it works if we use an import there

Reply:
I'm not sure exactly what to do here. If I do the opposite, TypeScript complains that the GridApi has properties that the EventEmitter instance doesn't support. Guidance welcomed.

Reply:
Ok we can pair up when you are free. Ping me on slack when it's a good time ;)

Reply:
Ok, let's do that tomorrow, for the sake of clarity, this is the current state of affair.

on master the type of the EventEmitter is `any`:

<img width="429" alt="Capture d’écran 2020-10-20 à 20 20 45" src="https://user-images.githubusercontent.com/3165635/96627971-e73f7700-1311-11eb-8904-874b0feeaff2.png">

and once I rewrite the EventEmitter, it fails on:

<img width="574" alt="Capture d’écran 2020-10-20 à 20 12 13" src="https://user-images.githubusercontent.com/3165635/96627996-ee668500-1311-11eb-85f5-0b887e4daa57.png">

which looks expected.

Reply:
Any idea?